File: Recursion/lecture 3/Fraz_recursion_3_2.py
def fibo(n):
  if n==1:
    return 1
  if(n==2):
    return 1
  else:
    partialAns1 = fibo(n-1)
    partialAns2 = fibo(n-2)
    return partialAns1+partialAns2
# Take input from user
n=int(input())
print(fibo(n))

File: Recursion/lecture 3/Fraz_recursion_3_2.cpp
#include <bits/stdc++.h>
using namespace std;

int fibo(int n)
{
    if (n == 1 || n == 2)
        return 1;
    int partialAns1 = fibo(n - 1);
    int partialAns2 = fibo(n - 2);
    return partialAns1 + partialAns2;
}

int main()
{
    int n;
    cin >> n;
    cout << fibo(n);
}
